nearlib/README.md

```diff
@@ -1,22 +1,90 @@
-Near lib
-====
+# Near lib
 
 Near lib is a javascript library for development of dapps on near.
 
 ## Install dependencies
-
 ```
 npm install
 ```
-
 ## Test
 
 Start the node by following instructions from nearcore/README.md, then
-
 ```
 npm test
 ```
-
 ## Integration test
 
-Contract "hello.wasm" source code location: https://studio.nearprotocol.com/?f=mfaccXt
+Contract "hello.wasm" source code location: <https://studio.nearprotocol.com/?f=mfaccXt>
+
+## documentation
+To regenerate documentation, run
+```
+documentation readme near.js signing/key_pair.js -f md --shallow --section=API
+```
+
+## API
```

Comment:
let's not copy this into README, just put a link. 
what we need in README is some simple code sample demonstrating main features.

Reply:
Strangely this tool's file output is different (and looks worse) than this API readme section version. I'll play around with this more to see if I can get it to output to file nicely, but for now this is the easiest way to get it to work.